ui/packages/consul-ui/vendor/consul-ui/services-debug.js

```diff
@@ -1,15 +1,16 @@
-(services => services({
-  "route:application": {
-    "class": "consul-ui/routing/application-debug"
-  },
-  "service:intl": {
-    "class": "consul-ui/services/i18n-debug"
-  }
-}))(
-  (json, data = document.currentScript.dataset) => {
-    const appNameJS = data.appName.split('-')
-      .map((item, i) => i ? `${item.substr(0, 1).toUpperCase()}${item.substr(1)}` : item)
-      .join('');
-    data[`${appNameJS}Services`] = JSON.stringify(json);
+(services =>
+  services({
+    'route:application': {
+      class: 'consul-ui/routing/application-debug',
+    },
+    'service:intl': {
+      class: 'consul-ui/services/i18n-debug',
+    },
+  }))(
+  (
+    json,
+    data = typeof document !== 'undefined' ? document.currentScript.dataset : module.exports
+  ) => {
+    data[`services`] = JSON.stringify(json);
```

Comment:
Development time only routes for:

1. application: This adds our docs menu/navigation if you go to anything under `/docs`
2. intl: various development utilities to help us convert the entire app to 100% support i18n (eventually! 😄 )